metricshub-snmpv3-extension/pom.xml

```diff
@@ -0,0 +1,247 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<parent>
+		<groupId>org.sentrysoftware</groupId>
+		<artifactId>metricshub-parent</artifactId>
+		<version>0.9.03-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>metricshub-snmpv3-extension</artifactId>
+
+	<name>MetricsHub SNMP V3 Extension</name>
+	<description>MetricsHub SNMP V3 Extension implementation providing a set of functionalities used to process SNMP criteria and sources.</description>
```

Comment:
`MetricsHub SNMP version 3 Extension implementation providing a set of functionalities used to process SNMP criteria and sources.`